src/Text/Pandoc/Readers/MediaWiki.hs

```diff
@@ -699,7 +699,7 @@ inlinesBetween start end =
 emph :: PandocMonad m => MWParser m Inlines
 emph = B.emph <$> inlinesBetween start end
     where start = sym "''"
-          end   = try $ notFollowedBy' (() <$ strong) >> sym "''"
+          end   = try $ notFollowedBy' (() <$ strong) >> ((sym "''") <|> void newline)
```

Comment:
Can you explain why you added the `<|> void newline` ?

Reply:
hah, this crept in because what I was _originally_ trying to do was address #3044 but then I noticed the `"wikilinks"` thing nearby and was distracted.

Reply:
OK, let's leave this out of this PR and address it separately.